pandas/core/frame.py

```diff
@@ -241,17 +241,52 @@ class DataFrame(NDFrame):
         Column labels to use for resulting frame. Will default to
         np.arange(n) if no column labels are provided
     dtype : dtype, default None
-        Data type to force, otherwise infer
+        Data type to force. Only a single dtype is allowed. If None, infer
```

Comment:
According to https://github.com/pandas-dev/pandas/issues/4464 and https://github.com/pandas-dev/pandas/issues/9133 it seems like this is not yet implemented.